### ISSUE TITLE
Installation issue

### ISSUE THREAD

**Issue by bharath-naik** [hpi-schul-cloud/jitsi-deployment#41]
I find this repo has installation issue related to metacontroller, there is no prerequisite information that has been provided. if someone has completed the installation with kustomize 3.5.4 Please share your steps, that may save lot of time. 
in my case pods are getting unscheduled and it has warnings.

**Comment by r3k2** [comment]
I agree, please add some documentation on this, I am pulling from diff other forums and diff solutions that will just make things worse
